packages/firestore/src/core/types.ts

```diff
@@ -47,9 +54,9 @@ export enum OnlineState {
   Healthy,
 
   /**
-   * The client considers itself offline. It is either trying to establish a
-   * connection but failing, or it has been explicitly marked offline via a call
-   * to disableNetwork().
+   * The client is either trying to establish a connection but failing, or it
+   * has been explicitly marked offline via a call to disableNetwork().
+   * Higher-level components should not operate in offline mode.
```

Comment:
I think this is the opposite of correct now. :-P

Reply:
Oops, yeah, cut-and-paste fail.